### PR TITLE
Feature/header css

### PR DESCRIPTION
**Pull request by fjmiguel**
Please review.

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -56,7 +56,7 @@ img{
 }
 
 ul{
-  list-style: none;
+  list-style: none111;
```
Comment (beteeliz) [inline assets/css/styles.css]: The value is none instead of "none111"

Comment (beteeliz) [reply assets/css/styles.css]: There is no .screen-sm-hidden {
display: none;
}
declared before the header

File: assets/css/styles.css [hunk 2]
```diff
@@ -98,8 +98,96 @@ place-items-center{
...
+  opacity: 1;
+  transform: scale(1);
+}
+
+.menu-toggle-icon.activated .open-menu-icon{
+  display: none;
+}
+
+.menu-toggle-icon.activated .close-menu-icon {
+  display: blocking2;
```
Comment (beteeliz) [inline assets/css/styles.css]: The display has the value block here instead of blocking2

File: assets/css/styles.css (hunk 1 above)
Comment (fjmiguel) [reply assets/css/styles.css]: about none111: already fixed. About the second point as we discussed this will be used later on. Tks.

**Review by fjmiguel — COMMENTED** [review]
Fixed. please check.

**Review by beteeliz — APPROVED** [review]
Approved!

**Pull request merged**